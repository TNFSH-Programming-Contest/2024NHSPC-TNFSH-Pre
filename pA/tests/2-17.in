861031734932770880 881981486185383460 966212048692007520
355019007991320445 816968371511440310 395186207076613935
