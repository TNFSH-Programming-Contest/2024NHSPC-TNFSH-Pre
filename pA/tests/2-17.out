374426939765966608
